tutorials/Tutorial 5 - Plotting.ipynb

```diff
@@ -11,12 +11,13 @@
   },
```

Comment:
<p>This should be <code>node_labels</code>,no?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ComplexGroupInteractions/xgi/pull/234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Right, I will correct it</p>